roles/logins/handlers/jumphost-quota.yml

```diff
@@ -0,0 +1,22 @@
+---
+#
+# Important: maintain correct handler order.
+# Handlers are executed in the order in which they are defined 
+# and not in the order in whch they are listed in a "notify: handler_name" statement!
+#
+- name: 'Rebuild GRUB config.'
+  shell: |
+         timestamp="$(date '+%Y-%m-%dT%H:%M:%S%z')"
```

Comment:
I'd have preferred a cp --backup for simplicity. (not breaking... but still)

Reply:
A `cp --backup` only works once and creates a copy suffixed with a tilde creating `original~`. If the playbook did not work and you simply try to run it a second time it will overwrite the `original~` again and then the original GRUB config is lost: both `original` and `original~` will be the same updated and possibly wrong code.

For the record: I'm not happy we need to update something as low level as the GRUB config to simply enable a mount option, but unfortunately this is required for quota on an XFS root partition. I've tried to make it as robust as possible, but the regexes in `tasks/jumphost-quota.yml` are not the easiest to read and I would not be surprised if this breaks on major updates :o. On the other hand such problems will be less frequent than users accidentally copying large files to the jumphost breaking logins for everyone when the jumphost runs out of disk space.

Reply:
ok, a backup --numbered then. It might not do <exactly> what you want but by choosing the path of lesser resistance, the readability improves dramatically.

Reply:
Ok,
`cp: unrecognized option '--numbered'`
--backup=numbered does work, but creates ugly suffixes and I like the explicit timestamps to assist in debugging when something went wrong. How about a --backup combined with a custom --suffix? Will add that in a new commit...

Reply:
Eh, never mind that also does not work. It would if the `grub2-mkconfig` command would support something like --backup. The cp is explicitly creating the backup copy here. I do not see a way to simply this.